__tests__/app.test.js

```diff
@@ -65,3 +65,52 @@ describe("GET /api/articles/:article_id", () => {
     expect(response.body.msg).toBe("Bad Request");
   });
 });
+
+describe("GET /api/articles", () => {
+  it("200: responds with all of the articles without body property and with comment_count property", async () => {
+    const response = await request(app).get("/api/articles");
+    expect(response.status).toBe(200);
+    expect(response.body.articles.length).toBe(13);
+    response.body.articles.map((article) => {
```

Comment:
This code isn't to do with the current ticket and so shouldn't be on this pr. However as a point of feedback on it, we shouldn't just be asserting against the keys, we need to assert against the values as well.